Bottom: 27423333f5d0263067f8bb07ea19d5601ac62d89
Top:    4256771355ddb701ff08b0a558dd501b9d655e61
Author: Peter Clifton (none) <peterc@paramita-desktop-2>
Date:   2011-05-11 12:07:15 +0100

hid/gtk: Fix get_coords function


---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 77a7416..e6e1e53 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -203,8 +203,8 @@ ghid_get_coords (const char *msg, int *x, int *y)
     ghid_get_user_xy (msg);
   if (ghid_port.has_entered)
     {
-      *x = SIDE_X (gport->view_x);
-      *y = SIDE_Y (gport->view_y);
+      *x = gport->view_x;
+      *y = gport->view_y;
     }
 }
